artemis/src/test/java/tech/pegasys/artemis/state/util/ValidatorsUtilTest.java

```diff
@@ -0,0 +1,290 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.artemis.state.util;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static tech.pegasys.artemis.Constants.ACTIVE_PENDING_EXIT;
+import static tech.pegasys.artemis.Constants.EXITED_WITHOUT_PENALTY;
+
+import tech.pegasys.artemis.datastructures.beaconchainstate.ValidatorRecord;
+import tech.pegasys.artemis.datastructures.beaconchainstate.Validators;
+import tech.pegasys.artemis.ethereum.core.Hash;
+import tech.pegasys.artemis.util.uint.UInt64;
+
+import java.util.ArrayList;
+
+import org.junit.Test;
+
+public class ValidatorsUtilTest {
+    public static double DOUBLE_ASSERTION_DELTA = 0.0d;
+
+    @Test
+    public void assert_Get_Zero_As_Effective_Balance_For_NullValidators() {
+        //given
+        double effective_Balance_Expected = 0.0d;
+
+        //when
+        double effective_Balance_Actual = ValidatorsUtil.get_effective_balance(null);
+
+        //then
+        assertEquals(effective_Balance_Expected, effective_Balance_Actual, DOUBLE_ASSERTION_DELTA);
+    }
+
+    @Test
+    public void assert_Get_Zero_As_Effective_Balance_For_Validators_With_SingleValidatorRecord_And_Zero_Balance() {
```

Comment:
There is significant duplicated setup code in the below methods. This should be refactored where possible.

Reply:
Thanks for review , schroedinger
I will make sure to :
- refactor the duplicated code in tests 
- Camel case standard changes in test and ValidatorsUtil
- update my editor settings to not to append white spaces